README.rst

```diff
@@ -1,13 +1,72 @@
 This package contains the Zephyr RTOS meta tool, 'west'.
 
-To install west from this package into your user directory, run:
+WARNING
+-------
 
-$ python3 setup.py install --user
+DO NOT INSTALL WITH "python3 setup.py install".
 
-You can also try it out by just running the package as a module from
-this directory:
+Please use pip to install in development mode as documented below.
+
+Important Note
+--------------
+
+West is distributed in two pieces:
+
+1. A bootstrap/wrapper script, which is distributed via PyPI.
+2. The "main" west package and entry points, which are fetched by the
+   bootstrap script.
+
+This somewhat unusual arrangement is because:
+
+- One of West's jobs is to manage interaction with Zephyr's multiple
+  Git repositories, including its own.
+- West is in its experimental stages and is moving quickly, meaning
+  users need to stay on HEAD.
+
+The default setup.py installs the **wrapper script only**.
+
+Installation
+------------
+
+To install the West bootstrapping/wrapper script from this package in
+development mode, clone this repository and run this from the top
+level directory:
+
+$ pip3 install -e .
+
+Then use the wrapper script to initialize a Zephyr installation with::
+
+  $ west init your/zephyr/install-dir
+  $ cd your/zephyr/install-dir
+  $ west <command>
+
+The ``west init`` call will:
+
+- create your/zephyr/install-dir
+- clone the Zephyr manifest repository (whose URL can be overridden
+  with the ``-u`` option, and branch with ``--mr`` / ``--manifest-rev``)
+- clone the latest West repository (the URL override is ``-w``, and
+  revision/branch override is ``--wr`` / ``--west-revision``)
```

Comment:
still `--west-revision` here, should it be `--west-rev`?

Reply:
-ETOOMUCHCOFFEE, thanks, fixed.